src/main/java/org/spongepowered/common/SpongeImplHooks.java

```diff
@@ -238,18 +234,13 @@ public static boolean canEntitySpawnHere(EntityLiving entityLiving, IEntityLivin
     }
 
     public static Object onUtilRunTask(FutureTask<?> task, Logger logger) {
-        for (WorldServer worldServer : WorldManager.getWorlds()) {
-            final CauseTracker otherCauseTracker = ((IMixinWorldServer) worldServer).getCauseTracker();
-            otherCauseTracker.switchToPhase(PluginPhase.State.SCHEDULED_TASK, PhaseContext.start()
-                    .add(NamedCause.source(task))
-                    .addCaptures()
-                    .complete()
-            );
-        }
+        GlobalCauseTracker.INSTANCE.switchToPhase(PluginPhase.State.SCHEDULED_TASK, PhaseContext.start()
```

Comment:
I updated it to use a getter method, but what do you mean by "sanity check"? I don't believe it can return null as the instance is automatically initialized.